src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -316,7 +316,7 @@ def k8s_install_connector(client, name, resource_group, connector_name,
                           chart_url=None, os_type='Linux'):
     from subprocess import PIPE, Popen
     helm_not_installed = "Helm not detected, please verify if it is installed."
-    image_tag = 'latest'
+    node_prefix = 'virtual-kubelet-' + connector_name.lower()
```

Comment:
Will this work if connector_name contains spaces or other special chars?
If not, handle this in _params.py where you can add a validator to that param.
Same for uninstall.